lib/ReactViews/Notification/NotificationWindow.jsx

```diff
@@ -65,14 +65,11 @@ const NotificationWindow = createReactClass({
       height: defined(this.props.height) ? this.props.height : "auto",
       width: defined(this.props.width) ? this.props.width : "500px"
     };
-    const isStory = type === "story";
 
     return (
       <div className={classNames(Styles.wrapper, `${type}`)}>
         <div
           className={Styles.notification}
-          /* eslint-disable-next-line react/no-unknown-property */
-          isStory={isStory}
```

Comment:
Does the comparison on line 78 work if you just remove this?

Reply:
thanks for catching this I restored the eslint-disable comments for those issues and will look in them more when this is merged into webpack5 PR